src/components/GamePage.vue

```diff
@@ -182,16 +218,33 @@ const KeyUp = () => {
   }
 };
 //ページ全体を開いている時にどこを押してもkeyeventが起こる
-document.onkeydown = (event) => KeyDown();
-document.onkeyup = (event) => KeyUp();
+document.onkeydown = (event) => {
+  if (user.canStartGame) {
+    KeyDown();
+  } else {
+    return;
+  }
```

Comment:
```suggestion
  if (user.canStartGame) {
    KeyDown();
  }
  return;
```
[IMO]
else文の中で特に何もしないのであれば、returnはいらないかなと思いました。
明示的にreturnさせたいのであれば、else文は消してもいいかな。